Model/Data/Quote.php

```diff
@@ -73,23 +73,19 @@ public function __construct(
      * @return array
      * @throws \Magento\Framework\Exception\InputException
      */
-    public function paymentDataFromQuote(MagentoQuote $quote, $installmentsCounts = 3): array
+    public function paymentDataFromQuote(MagentoQuote $quote, $installmentsQuery): array
```

Comment:
update the docblock `@param` to match parameters

Reply:
this is actually `eligibilityDataFromQuote` (because paymentData have another format now) : refactor method name ?